CesiumUtility/src/Uri.cpp

```diff
@@ -10,20 +10,35 @@
 #include <vector>
 
 namespace CesiumUtility {
+const char* HTTPS_PREFIX = "https";
+const char* HTTPS_PREFIX_COLON = "https:";
+
+std::string cesiumConformUrl(const std::string& url) {
+  if (url.rfind("//", 0) == 0) {
+    return std::string(HTTPS_PREFIX_COLON).append(url);
+  } else if (url.find("://", 0) == 0) {
+    return std::string(HTTPS_PREFIX).append(url);
+  }
```

Comment:
Could probably save some performance here by checking if the char at index 1 is not a '/', just return the url. As it stands, a non-protocol-relative url executes both finds when passed into here.

Come to think of it, you could probably just check the chars at indices 0, 1, 2 directly without doing any finds at all.

Reply:
Very true! This was actually supposed to be `rfind` at index 0, which would make a single comparison with the start of the string, but I accidentally rewrote it to `find` some point while working on it. Though, I've also realized we don't need to check the `://` case, since despite what I remembered, this isn't actually a valid form of a protocol-relative URI.